### PR TITLE
fix(#335)

### PR DESCRIPTION
**Pull request by neoFelhz**
**Contributing rules**

- Fork the repo and create your branch from `canary`. Then be sure to put the `canary` branch as the target for your pull request.
- Please be sure to follow the [contributing guidelines](https://github.com/viosey/hexo-theme-material/blob/master/CONTRIBUTING.md), especially for commit message
- Remove the `Contributing rules` part from this description
- Fill out the other parts from this description

> If you don't do so, we might change your pull request's title and using `squash` to merge your changed.

**What kind of change does this PR introduce?** (check one with "x")

- [x] Bug fix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

____

**Description**

No description.

____

**Verification steps**

No verification steps.

### CODE REVIEW

File: source/css/style.css [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ a {
 @media screen and (max-width: 840px) {
     .mdl-mini-footer--right-section {
         margin-left: auto;
-        margin-right: auto;
+        margin-right: auto!important;
```
Comment (cubesky) [inline source/css/style.css]: This fix is so ugly....

Comment (neoFelhz) [inline source/css/style.css]: Well, show me your code. : )

File: source/css/style.min.css [hunk 2]
```diff
@@ -1 +1 @@
...
...nopic a,#scheme-Paradox .article-headline-p-top a,#scheme-Paradox .index_category-link,#scheme-Paradox .index_share-link,#scheme-Paradox .material-nav a,#scheme-Paradox .post_share-link,#scheme-Paradox .post_tag-link,#scheme-Paradox .sidebar .sidebar-brand,#scheme-Paradox .sidebar-footer-text-a{text-decoration:none}#scheme-Paradox .index_category-link,#scheme-Paradox .post_tag-link{padding:15px}#scheme-Paradox .mdl-mini-footer--right-section{margin-right:60px}@media screen and (max-width:480px){#scheme-Paradox .mdl-mini-footer--right-section{margin-right:0}}#scheme-Paradox .mdl-mini-footer--right-section a{color:#6b6b6b}#scheme-Paradox #author-avatar,#scheme-Paradox #comment__avatar{margin-right:1pc;border-radius:50%;overflow:hidden}#scheme-Paradox .mdl-mini-footer--social-btn{outline-style:none}#scheme-Paradox .material-nav{font-weight:300!important}#scheme-Paradox .material-nav a{font-weight:400}#scheme-Paradox .login-form-group{width:33%;float:left;color:#8a8a8a}#scheme-Paradox .login-input-info{font-size:14px!important}#scheme-Paradox .comment__text{font-weight:400!important;font-size:13px!important;padding-left:4pc}#scheme-Paradox .comment__text p{margin-bottom:6px}#scheme-Par
```
Comment (viosey) [inline source/css/style.min.css]: need minified again

Comment (neoFelhz) [inline source/css/style.min.css]: https://github.com/viosey/hexo-theme-material/pull/358/commits/38494d89f0c70a3a0b714bb9114082e2ac17c071

**Pull request merged**